tasks/68_drop-elements/drop-elements.js

```diff
@@ -0,0 +1,14 @@
+export const dropElements = (arr, fn) => {
+  let passed = arr.findIndex(fn);
```

Comment:
If we are finding an index like this, do we need the while that is just shifting elements?